ERCS/erc-8002.md

```diff
@@ -0,0 +1,291 @@
+---
+eip: 8002
+title: Simplified Payment Verification Gateway
+description: Trustless singleton contract for on-chain verification of Bitcoin transactions through block headers
```

Comment:
Whenever there's a singleton contract, we have to ask the question: does it need an ERC?

If there's only one deployment, would it make more sense to document it on the project's page? ERCs really shine when multiple implementations need to coordinate.

That said, we do have other singletons documented here, so if your reasoning is sound, we'll include it.